Fix x86 build failure: only use MULTIARCH_PATH if it's defined :-)

	--xtraeme

--- a/pcc/cc/cc/cc.c	2014-12-18 14:57:43.961103113 +0100
+++ b/pcc/cc/cc/cc.c	2014-12-18 14:57:52.881207162 +0100
@@ -207,9 +207,9 @@ char *cppmdadd[] = CPPMDADD;
 #endif
 #ifndef DEFLIBDIRS	/* default library search paths */
 #ifdef MULTIARCH_PATH
-#define DEFLIBDIRS	{ "/usr/lib/", 0 }
-#else
 #define DEFLIBDIRS	{ "/usr/lib/", "/usr/lib/" MULTIARCH_PATH "/", 0 }
+#else
+#define DEFLIBDIRS	{ "/usr/lib/", 0 }
 #endif
 #endif
 #ifndef DEFLIBS		/* default libraries included */
